common/supplemental/supplementalData.xml

```diff
@@ -1315,7 +1314,6 @@ XXX Code for transations where no currency is involved
 		<language type="arp" scripts="Latn"/>
 		<language type="arq" scripts="Arab"/>
 		<language type="arq" territories="DZ" alt="secondary"/>
-		<language type="ars" scripts="Arab" territories="SA"/>
```

Comment:
I'll have to look into this one.

Reply:
The line is gone because it doesn't meet the threshhold for inclusion in the basic language data.  The criteria are as follows:


```
            if (rowData.officialStatus.isMajor()) {
                status_territories.put(BasicLanguageData.Type.primary, rowData.countryCode);
            } else if (rowData.officialStatus.isOfficial()
                || rowData.getLanguagePopulation() >= cutoff * rowData.countryPopulation
                || rowData.getLanguagePopulation() >= 1000000) {
                status_territories.put(BasicLanguageData.Type.secondary, rowData.countryCode);
            }
        }
```

Cutoff here is 20%, so basically it's 1,000,000 speakers or at least 20% of population.